docs/sources/developers/plugins/development-with-local-grafana.md

```diff
@@ -0,0 +1,89 @@
++++
+title = "Development with local Grafana"
++++
+
+# Development with local Grafana
+
+This guide allows you to setup a development environment where you run Grafana and your plugin locally. With this, you will be able to see your changes as you add them.
+
+## Run Grafana in your host
+
+If you have git, Go and the required version of NodeJS in your system, you can clone and run Grafana locally:
+
+1. Clone [Grafana](https://github.com/grafana/grafana)
+
+   a. [optional] Checkout to the specific version you want to target (e.g. `git checkout v8.3.4`)
+
+2. Install its dependencies
+
+   ```bash
+   yarn install
+   ```
+
+3. Grafana will look for plugins, by default, on its `data/plugins` directory. You can create a symbolic link to your plugin repository to detect new changes:
+
+   ```bash
+   ln -s <plugin-path>/dist data/plugins/<plugin-name>
+   ```
+
+4. (Optional) If the step above doesn't work for you (e.g. you are running on Windows), you can also modify the default path in the Grafana configuration (that can be found at `conf/custom.ini`) and point to the directory with your plugin:
+
+   ```ini
+   [paths]
+   plugins = <path-to-your-plugin-parent-directory>
+   ```
+
+## Run Grafana with docker-compose
```

Comment:
Not sure what value it could bring here but we're currently making efforts to update the [plugin examples](https://github.com/grafana/grafana-plugin-examples) to include [docker-compose dev env](https://github.com/grafana/grafana-plugin-examples/tree/master/examples/datasource-basic) similar to the instructions found here.

Reply:
nice! let me add a note here so people using newer version of the examples don't need to copy this file.

I have also modified this docker-compose to be closer to that one